t/init.vim

```diff
@@ -55,6 +55,14 @@ describe 'init sections'
     Expect g:airline_section_z =~ '%v'
   end
 
+  it 'section gutter should be csv'
```

Comment:
that comment looks wrong

Reply:
@chrisbra 

Could you see this document? 
> https://github.com/vim-airline/vim-airline/blob/master/doc/airline.txt#L409

If this test comment is wrong, I think we need to fix this line in the documentation as well. How do you think this should be fixed?

Reply:
ah, the thing is, this should be empty, unless the `csv` extension is also installed.